cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,24 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('User', () => {
-  before(() => {
+import UserPageObject from '../support/pages/user.pageObject';
+
+const userPage = new UserPageObject();
 
+describe('User', () => {
+  let user;
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.register();
   });
 
   it.skip('should be able to follow the another user', () => {
-
+    cy.createUser(user.email, user.username);
```

Comment:
It seems that you are creating a user twice, both on line 18 and 22. Since user is already created with cy.register(), you can remove this line.